tests-integration/phd-client.integration.ts

```diff
@@ -0,0 +1,215 @@
+import { expect, sinon } from '@loopback/testlab'
```

Comment:
this is just a nit-pick but in loopback terms, acceptance tests == integration tests. So you could move this file and/or folder within the acceptance folder so it goes more like the loopback's recommended folder structure

Reply:
Makes sense. Actually the only reason I put these tests there was to have an easier time running them on their own but I have no objection to moving them wherever you feel is best.

Reply:
all good, going with the recommended way then is the one I mentioned above

Reply:
Done. PHD Client integration tests moved to `tests/acceptance`